docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutSlots.js

```diff
@@ -124,7 +139,9 @@ function DashboardLayoutSlots(props) {
       theme={demoTheme}
       window={demoWindow}
     >
-      <DashboardLayout slots={{ toolbarActions: Search }}>
+      <DashboardLayout
+        slots={{ toolbarActions: Search, sidebarFooter: SidebarFooter }}
```

Comment:
I wonder about naming. Should we call this something like "NavigationFooter"? 🤔  I feel like it would make sense to give our surfaces a semantic name, rather one that is based on where it's positioned. Not sure though

Reply:
So far I've used the word "sidebar" for this kind of interfaces and it made the most sense to me in the places it was used, where it seemed like they were specific features of the sidebar itself and not just navigation.
The `hideNavigation` prop in `DashboardLayout` is an exception but it also hides the menu icon, so it's a bit more general.
For this particular slot I think we really mean the footer of the sidebar and no other type of navigation so I feel like it makes sense to name it this way? And it might be easier for users to understand what it really means visually.

It's also just my opinion of course, but the logic I've been following for this kind of naming is intentional.